pagopa/io-web-profile
# [IOPID-1558] : Email Validation - Confirmed Page

## Short description

This pull request aims to implement the email-validation **email confirmed page** within the io web profile project, as outlined in the JIRA task.

## List of changes proposed in this pull request

- Added deeplink (back to io) util function and env url
- Added Email Confirmed Page
- Updated MockFile according to the spec openapi

## How to test

1. To enable the email validation page, set the env variable **NEXT_PUBLIC_VALIDATION_EMAIL to true**
2. Enable (update) Mockoon with the new mock (stored in the mock folder).
3. Access the email confirmation page using the provided token in the query parameter and verify API calls according to OpenAPI specifications.

    Example URL: http://localhost:3000/it/conferma-email/?token=05QSY3JXN8XF47LTKRW9EMHZBX:179aeae8dcc01abdab31e5ba

4. Click on Confirm Button, you should see the confirmation page (if we get 200)

## Video
[IOPID-1558.webm](https://github.com/pagopa/io-web-profile/assets/145718190/6fed6d97-b0bf-4e45-89a0-a08e25ade1ed)
